### PR TITLE
Updated tests with lws number change, and tidied up some redundant fi…

### PR DESCRIPTION
**Pull request by KieranWardle**
…elds

### CODE REVIEW

File: src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/CommonSteps.java [hunk 1]
```diff
@@ -109,6 +109,11 @@ public void we_ingest_them() throws IOException {
 
     fsdrUtils.ingestAdecco();
     fsdrUtils.ingestRunFSDRProcess();
+    try {
+      Thread.sleep(5000);
```
Comment (garethTurner) [inline src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/CommonSteps.java:113]: why is this needed?

File: src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/LwsSteps.java [hunk 2]
```diff
@@ -151,13 +152,13 @@ public void the_employee_in_the_LWS_CSV_as_an_mover(String id, String roleId) {
...
     assertThat(record).contains("\"takeOnCode\":\"ONSTAKEonLIVE\"");
-    assertThat(record).contains("\"phoneNumber\":\"0123456789\"");
+    assertThat(record).contains("\"phoneNumber\":\"4423456789\"");
     assertThat(record).contains("\"personalMobileNumber\":\"0987654321\"");
     assertThat(record).contains("\"pinNumber\":-2");
     assertThat(record).contains("\"updateMode\":\"BULKLOADER\"");
     assertThat(record).contains("\"templatePersonId\":-1");
     assertThat(record).contains("\"newPersonId\":-1");
-    assertThat(record).contains("\"operatorInstructions1\":\"Contact Lone Worker on mobile: 0123456789\"");
+    assertThat(record).contains("\"operatorInstructions1\":\"Contact Lone Worker on mobile: 0723456789\"");
```
Comment (garethTurner) [inline src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/LwsSteps.java:161]: any idea if these numbers should be in the same format?

File: src/test/java/uk/gov/ons/fsdr/tests/acceptance/utils/QueueClient.java [hunk 3]
```diff
@@ -62,8 +60,9 @@ public void clearQueues() throws URISyntaxException {
       clearQueue("Xma.Events");
       clearQueue("Xma.FieldOfficer");
       clearQueue("Xma.Leaver");
-      clearQueue("xma.transient.error");
-      clearQueue("report.events");    }
+     // clearQueue("xma.transient.error");
```
Comment (garethTurner) [inline src/test/java/uk/gov/ons/fsdr/tests/acceptance/utils/QueueClient.java:63]: should this be commented out?

File: src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/CommonSteps.java (hunk 1 above)
Comment (KieranWardle) [reply src/test/java/uk/gov/ons/fsdr/tests/acceptance/steps/CommonSteps.java:113]: Oops didn't mean to leave that there
